pyomo/repn/plugins/baron_writer.py

```diff
@@ -256,23 +256,24 @@ def _skip_trivial(constraint_data):
             suffix_gen = (
                 lambda b: pyomo.core.base.suffix.active_export_suffix_generator(b)
             )
-        r_o_eqns = []
-        c_eqns = []
-        l_eqns = []
+        r_o_eqns = {}
+        c_eqns = {}
+        l_eqns = {}
```

Comment:
I know this isn't an artifact of this PR, but what do `r_o`, `c`, and `l` stand for?

Reply:
See https://github.com/Pyomo/pyomo/blob/3f6bc13fdb6e9f1f772ca6401b59ab4e08be180d/pyomo/repn/plugins/baron_writer.py#L270-L277
- `r_o`: `relaxationonly`
- `c`: `convex`
- `l`: `local`